frontend/src/queries/nodes/InsightViz/EditorFilters.tsx

```diff
@@ -287,78 +290,58 @@ export function EditorFilters({ query, showing, embedded }: EditorFiltersProps):
         },
         {
             title: 'Advanced Options',
-            editorFilters: filterFalsy([
-                isPaths && {
-                    key: 'paths-advanced',
-                    position: 'left',
-                    component: PathsAdvanced,
-                },
-                isFunnels && {
-                    key: 'funnels-advanced',
-                    position: 'left',
-                    component: FunnelsAdvanced,
-                },
-            ]),
-        },
-        {
-            title: 'PoE Override',
+            defaultExpanded: false,
             editorFilters: filterFalsy([
                 {
                     key: 'poe',
-                    position: 'right',
                     component: PoeFilter,
                 },
-            ]),
-        },
-        {
-            title: 'Sampling',
-            editorFilters: filterFalsy([
                 {
                     key: 'sampling',
-                    position: 'right',
                     component: SamplingFilter,
                 },
+                isTrends && {
```

Comment:
IMO this should only appear for certain visualizations, as some (like number, pie chart) don't make sense

Reply:
That makes sense. I meant to do this and ended up forgetting, on it